<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="1" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-04-02T18:18:23 IST" name="InetBankingV1" finished-at="2022-04-02T18:18:40 IST" duration-ms="16357">
    <groups>
    </groups>
    <test started-at="2022-04-02T18:18:23 IST" name="BankingAppTest" finished-at="2022-04-02T18:18:40 IST" duration-ms="16357">
      <class name="com.inetBankingV1.testCases.TC_LoginTest_DDT_002">
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:com.inetBankingV1.testCases.TC_LoginTest_DDT_002@68dc098b]" started-at="2022-04-02T18:18:23 IST" name="setup" finished-at="2022-04-02T18:18:30 IST" duration-ms="7251" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="LoginTestDDT(java.lang.String, java.lang.String)[pri:0, instance:com.inetBankingV1.testCases.TC_LoginTest_DDT_002@68dc098b]" started-at="2022-04-02T18:18:31 IST" name="LoginTestDDT" data-provider="getData" finished-at="2022-04-02T18:18:35 IST" duration-ms="3605" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr392805]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[uvYmuzA]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginTestDDT -->
        <test-method signature="LoginTestDDT(java.lang.String, java.lang.String)[pri:0, instance:com.inetBankingV1.testCases.TC_LoginTest_DDT_002@68dc098b]" started-at="2022-04-02T18:18:35 IST" name="LoginTestDDT" data-provider="getData" finished-at="2022-04-02T18:18:36 IST" duration-ms="1915" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr380567]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[bajUsav]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertTrue(Assert.java:52)
at com.inetBankingV1.testCases.TC_LoginTest_DDT_002.LoginTestDDT(TC_LoginTest_DDT_002.java:35)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginTestDDT -->
        <test-method signature="LoginTestDDT(java.lang.String, java.lang.String)[pri:0, instance:com.inetBankingV1.testCases.TC_LoginTest_DDT_002@68dc098b]" started-at="2022-04-02T18:18:37 IST" name="LoginTestDDT" data-provider="getData" finished-at="2022-04-02T18:18:39 IST" duration-ms="1786" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[dngr480547]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[7ajUsap]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertTrue(Assert.java:52)
at com.inetBankingV1.testCases.TC_LoginTest_DDT_002.LoginTestDDT(TC_LoginTest_DDT_002.java:35)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginTestDDT -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.inetBankingV1.testCases.TC_LoginTest_DDT_002@68dc098b]" started-at="2022-04-02T18:18:39 IST" name="tearDown" finished-at="2022-04-02T18:18:40 IST" duration-ms="643" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.inetBankingV1.testCases.TC_LoginTest_DDT_002 -->
    </test> <!-- BankingAppTest -->
  </suite> <!-- InetBankingV1 -->
</testng-results>
